site/content/how-to/monitoring/tracing.md

```diff
@@ -0,0 +1,335 @@
+  - "cafe.example.com"
+  rules:
+  - matches:
+    - path:
+        type: PathPrefix
+        value: /coffee
+    backendRefs:
+    - name: coffee
+      port: 80
+EOF
+```
+
+Let's ensure that traffic can flow to our application.
+
+{{< note >}}If you have a DNS record allocated for `cafe.example.com`, you can send the request directly to that hostname, without needing to resolve.{{< /note >}}
+
+```shell
+curl --resolve cafe.example.com:$GW_PORT:$GW_IP http://cafe.example.com:$GW_PORT/coffee
+```
+
+We should see a response from the coffee Pod.
+
+```text
+Server address: 10.244.0.69:8080
+Server name: coffee-6b8b6d6486-k5w5w
+URI: /coffee
+```
+
+Assuming that you have access to the [Jaeger dashboard](http://127.0.0.1:16686) from earlier in the guide, you shouldn't see any tracing information yet. This means we need to create our `ObservabilityPolicy`.
+
+### Create the ObservabilityPolicy
+
+To enable tracing for our coffee HTTPRoute, we create the following policy:
+
+```yaml
+kubectl apply -f - <<EOF
+apiVersion: gateway.nginx.org/v1alpha1
+kind: ObservabilityPolicy
+metadata:
+  name: coffee
+spec:
+  targetRefs:
+  - group: gateway.networking.k8s.io
+    kind: HTTPRoute
+    name: coffee
+  tracing:
+    strategy: ratio
+    ratio: 75
+    spanAttributes:
+    - key: coffee-key
+      value: coffee-value
+EOF
+```
+
+This policy attaches to the coffee HTTPRoute and enables ratio-based tracing, where 75% of requests will be sampled. We've also included a span attribute to add extra data to the spans.
+
+Let's check the status of the policy:
+
+```shell
+kubectl describe observabilitypolicies.gateway.nginx.org coffee
+```
+
+```text
+Status:
+  Ancestors:
+    Ancestor Ref:
+      Group:      gateway.networking.k8s.io
+      Kind:       HTTPRoute
+      Name:       coffee
+      Namespace:  default
+    Conditions:
+      Last Transition Time:  2024-05-23T18:13:03Z
+      Message:               Policy is accepted
+      Observed Generation:   1
+      Reason:                Accepted
+      Status:                True
+      Type:                  Accepted
+    Controller Name:         gateway.nginx.org/nginx-gateway-controller
+```
+
+The policy is accepted, so now let's send some more traffic. Run the following command multiple times.
+
+```shell
+curl --resolve cafe.example.com:$GW_PORT:$GW_IP http://cafe.example.com:$GW_PORT/coffee
+```
+
+Once complete, let's refresh the Jaeger dashboard. We should now see a service entry called `ngf:default:cafe`, and a few traces. The service name by default is `ngf:<gateway-namespace>:<gateway-name>`.
+
+{{<img src="img/jaeger-trace-overview.png" alt="">}}
+
+<br></br>
+
+If we click into one of the traces, we can see the attributes.
+
+{{<img src="img/jaeger-trace-attributes.png" alt="">}}
+
+As you can see, the trace includes the attribute from the global NginxProxy resource, set by the cluster operator, as well as the attribute from the ObservabilityPolicy, set by the application developer.
```

Comment:
```suggestion
The trace includes the attribute from the global NginxProxy resource as well as the attribute from the ObservabilityPolicy.
```

I've removed mention of user personas because they do not appear anywhere else in this documentation, and in context, the reader has assumed both roles. The contemporary approach for writing docs is task-oriented use cases that don't presume a specific persona or role for the person running through them.